archeryutils/handicaps/handicap_equations.py

```diff
@@ -12,8 +12,6 @@
 import json
 from dataclasses import dataclass
 
-from archerycls import targets, rounds
```

Comment:
If removing this line do the docstrings need changing to find the correct types for Target, Round, and Pass?

Reply:
I'm honestly not sure. One good excuse to keep them would be to add type hinting to the function itself. I'll get on that in a minute...